src/index.css

```
@@ -651,18 +746,192 @@ section {
+/* Soft animated gradient halo behind the photo */
+.orbit-glow {
+  position: absolute;
+  inset: 12%;
+  border-radius: 9999px;
+  background: var(--accent-gradient);
+  filter: blur(38px);
+  opacity: 0.45;
+  z-index: 1;
+  animation: orbitGlowPulse 6s ease-in-out infinite;
+}
+@keyframes orbitGlowPulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "orbitGlowPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)